### ISSUE TITLE
18 and 34 layer configuration

### ISSUE THREAD

**Issue by kaishijeng** [MichaelHunson/ResNet_with_IdentityMapping#1]
Do you have a configuration for 18 and 34 layers?

Thanks

**Comment by yuhanghe01** [comment]
Yes, I just released the code on my profile. Besides, I also modified 50/101/152 layer generation file.

**Comment by kaishijeng** [comment]
Why is # of output in classifier layer 7, not 1000?
For IMageNet 2012, it should be 1000

**Comment by yuhanghe01** [comment]
You can definitely alert it to 1000, according to your task.
